pkg/session/session.go

```diff
@@ -3212,7 +3212,7 @@ func splitAndScatterTable(store kv.Storage, tableIDs []int64) {
 		for _, id := range tableIDs {
 			regionIDs = append(regionIDs, ddl.SplitRecordRegion(ctxWithTimeout, s, id, id, variable.DefTiDBScatterRegion))
 		}
-		if variable.DefTiDBScatterRegion {
+		if variable.DefTiDBScatterRegion != "" {
```

Comment:
Suggest use enum type to represent value list。

Reply:
I say the enum type value is only use in package level, here we still use string directly.